pkg/terraform/terraform.go

```diff
@@ -127,3 +134,30 @@ func unpackAndInit(dir string, platform string) (err error) {
 	}
 	return nil
 }
+
+func setupEmbeddedPlugins(dir string) error {
+	execPath, err := os.Executable()
+	if err != nil {
+		return errors.Wrapf(err, "failed to find path for binary %q", "openshift-install")
```

Comment:
nit: `"openshift-install"` -> `os.Args[0]`.  Users are free to rename our executable if they want ;).  I'm fine falling back to `openshift-install` if `len(os.Args) == 0` (which [can happen in POSIX][1], although I dunno if Go injects a default or not).

[1]: https://pubs.opengroup.org/onlinepubs/9699919799/functions/exec.html

Reply:
what about just `failed to find path for binary` ?

Reply:
done with https://github.com/openshift/installer/pull/919/commits/8f4946310428d4f1ade314b1d734be820dd66495